packages/stryker-mocha-runner/src/MochaOptionsLoader.ts

```diff
@@ -0,0 +1,75 @@
+import * as path from 'path';
+import * as fs from 'fs';
+import { StrykerOptions } from 'stryker-api/core';
+import { getLogger } from 'log4js';
+import MochaRunnerOptions, { mochaOptionsKey } from './MochaRunnerOptions';
+
+export default class MochaOptionsLoader {
+
+  private log = getLogger(MochaOptionsLoader.name);
+
+  load(config: StrykerOptions): MochaRunnerOptions {
+    const mochaOptions = Object.assign({}, config[mochaOptionsKey]) as MochaRunnerOptions;
+    if (mochaOptions && mochaOptions.opts) {
+      const optsFileName = path.resolve(mochaOptions.opts);
+      this.log.info(`Loading mochaOpts from "${optsFileName}"`);
+      const options = fs.readFileSync(optsFileName, 'utf8');
+      return Object.assign(this.parseOptsFile(options), mochaOptions);
+    } else {
+      this.log.debug('No mocha opts file specified, not loading additional mocha options (%s.opts was not defined).', mochaOptionsKey);
+      return mochaOptions;
+    }
+  }
+
+  private parseOptsFile(optsFileContent: string): MochaRunnerOptions {
+    const options = optsFileContent.split('\n').map(val => val.trim());
+    const mochaRunnerOptions: MochaRunnerOptions = Object.create(null);
+    options.forEach(option => {
+      const args = option.split(' ').filter(Boolean);
+      if (args[0]) {
+        switch (args[0]) {
+          case '--require':
```

Comment:
Doesn't `case '--require' || '-r':` work?

Reply:
Nope. Apparently not (tried it)